README.md

```diff
@@ -49,6 +49,7 @@ The mediator is especially useful when the edge entities are not always online,
 ## Protocols
 - [DONE] `BasicMessage 2.0` - https://didcomm.org/basicmessage/2.0
 - [DONE] `MediatorCoordination 2.0` - https://didcomm.org/mediator-coordination/2.0
+ - See [link for the protocol specs](/Coordinate-Mediation-Protocol.md)
```

Comment:
This file was copied from BB repo.
Would be nice to create detailed documentation for the other protocols used by the mediator.